tests/test_alias_dict.cpp

```diff
@@ -421,17 +421,6 @@ TEST_F(AliasDictionary, get_unexisting_alias_returns_nullopt)
     ASSERT_EQ(dict.get_alias("unexisting"), std::nullopt);
 }
 
-TEST_F(AliasDictionary, creates_backup_db)
-{
-    populate_db_file(AliasesVector{{"some_alias", {"some_instance", "some_command", "map"}}});
-
-    QString bak_filename = QString::fromStdString(db_filename() + ".bak");
-    ASSERT_FALSE(QFile::exists(bak_filename));
-
-    populate_db_file(AliasesVector{{"another_alias", {"an_instance", "a_command", "map"}}});
-    ASSERT_TRUE(QFile::exists(bak_filename));
-}
-
```

Comment:
Oh, I had missed this comment. This tested that the `AliasDictionary` created a backup file to rollback to on error. That functionality was replaced with `QSaveFile` in `write_json`, because: a) I couldn't make it work with the existing `QTemporaryFile` implementation (QSaveFile itself uses the same temporary file approach that the alias dict and snapshots used to make it transational and it turns out that it doesn't work well when the target is itself a temporary file); b) the current write_json implementation already does everything that was implemented on a case-by-case basis before, including the alias dictionary.